index.js

```diff
@@ -0,0 +1,53 @@
+let bookCollection = localStorage.getItem('bookCollection')
+  ? JSON.parse(localStorage.getItem('bookCollection'))
+  : [];
+const title = document.querySelector('#title-input');
+const author = document.querySelector('#author-input');
+const bookContainer = document.querySelector('.books-container');
+
+const removeBook = function () {
```

Comment:
- [ OPTIONAL ] You have a warning here (`Unexpected unnamed function.`). I would suggest you to name the function and remove the warning